byway/src/main/webapp/routepage.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="UTF-8">
+  <title>Route Generator</title>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" 
+    integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
+  <script src="https://polyfill.io/v3/polyfill.min.js?features=default"></script>
+  <script
+      src="https://maps.googleapis.com/maps/api/js?key=AIzaSyBg6D1U0U2L7VlbKH7Bkh-ds6wdUaLzroM&callback=initMap&libraries=&v=weekly"
+      defer></script>
+  <script src="script.js"></script>
+  <link rel="stylesheet" href="style.css">
+</head>
+<body onLoad="getRecs()">
+  <p>Place holder button to draw route</p>
+  <button id = "route">Generate Route</button>
+  <div class = "route-container">
+    <div class = "map-style">
+      <div id="map"></div>
+    </div>
+    <div class = "column">
+      <div class="box-outline">
+        <div class= "title" id="schedule">
+          <h3>Schedule</h3>
+        </div>
+       <div class="center" id="stop-list"></div>
+       </div>
+       <div class="time-outline">
+        <div class = "time-title">
+          <p>Estimated Travel Time:</p>
+        </div>
+        <div class="time-field">
+          <p>1 hr 30 mins</p>
+        </div>
+      </div>
+    </div>
+    
+    <div class="box-outline" id="box-outline-stops">
+      <div class= "title" id="stops">
+        <h3>Recommended Stops</h3>
+      </div>
+    <div class="center" id="rec-list"></div>
+  </div>
+  
+  <script src="https://code.jquery.com/jquery-3.1.1.min.js"></script>
```

Comment:
Not sure what this stuff is for - there doesn't seem to be any jQuery stuff

Reply:
This is required plugin for bootstrap, which I have used a tiny bit for containers, formatting (i.e. column)